_includes/tutorials/ksql-nested-json/ksql/markup/dev/problem-description.adoc

```diff
@@ -0,0 +1,34 @@
+Let's say you have a topic `financial_txns`, containing stock purchase at your firm. You want to do some basic reporting that includes the the number of shares, customer-id, and the ticker symbol involved in the transaction. All the information is in the record, but it's in JSON format and the information you need is nested.  Here's an example record:
```

Comment:
Not sure if this is overkill or not, but I thought it might be good to be explicit about the JSON structure issue.

Reply:
I don't think this is overkill, I like that you are visually calling out what data you need out of the event.

Reply:
One thing you should note is the number of nested fields is consistent across multiple events. Robin recently wrote a [blog](https://rmoff.net/2020/10/07/ksqldb-how-to-model-a-variable-number-of-fields-in-a-nested-value-struct/) about how you need to use `MAP` to model nested fields with a variable number of fields. 
I'm not sure if you would want to go into detail in this KT of how to use `MAP` and when.

Reply:
@awalther28 good call.  I think it's best for a KT only to cover one subject at a time.  But I've added a small section discussing this point and link out to Robin's post.  I've also [created an issue](https://github.com/confluentinc/kafka-tutorials/issues/634) for a separate KT on this subject.